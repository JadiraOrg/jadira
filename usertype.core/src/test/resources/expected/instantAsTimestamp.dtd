<!ELEMENT dataset (
    INSTANTASTIMESTAMP*)>

<!ELEMENT INSTANTASTIMESTAMP EMPTY>
<!ATTLIST INSTANTASTIMESTAMP
    ID CDATA #REQUIRED
    INSTANT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

