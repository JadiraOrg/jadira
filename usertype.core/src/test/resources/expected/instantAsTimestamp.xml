<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "instantAsTimestamp.dtd">
<dataset>
  <instantAsTimestamp ID="0" INSTANT="2004-02-25 17:03:45.76" NAME="test_0"/>
  <instantAsTimestamp ID="1" INSTANT="1980-03-11 00:03:45.0" NAME="test_1"/>
</dataset>